dotcom-rendering/src/components/marketing/banners/ThreeTierChoiceCardsV2.tsx

```diff
@@ -0,0 +1,147 @@
+import { css } from '@emotion/react';
+import {
+	palette,
+	space,
+	textSans12,
+	textSans15,
+	textSansBold14,
+	textSansBold20,
+} from '@guardian/source/foundations';
+import { LinkButton, SvgTickRound } from '@guardian/source/react-components';
+
+const RecommendedPill = () => {
+	return <div css={styles.recommendedPillStyles}>Recommended</div>;
+};
+
+export const ThreeTierChoiceCardsV2 = () => {
+	return (
+		<div css={styles.container}>
+			<div css={styles.choiceCardStyles}>
+				<div css={styles.greyHeadline}>Support just once</div>
+				<div css={styles.priceHeadline}>From just £1</div>
+				<div>
+					<LinkButton
+						href={'https://support.theguardian.com/contribute'} // URL will need to be updated
+						cssOverrides={styles.linkButtonStyles}
+					>
+						Continue
+					</LinkButton>
+				</div>
+				<div css={styles.benefitsStyles}>
+					We welcome support of any size, any time - whether you
+					choose to give £1 or more.
+				</div>
+			</div>
+			<div>
+				<div css={styles.choiceCardStyles}>
+					<RecommendedPill />
+					<div css={styles.greyHeadline}>Support monthly</div>
+					<div css={styles.priceHeadline}>£12/month</div>
+					<div>
+						<LinkButton
+							href={'https://support.theguardian.com/contribute'} // URL will need to be updated
```

Comment:
This will be updated before this test is launched

Reply:
card to make change to URLs https://trello.com/c/84LDsJsG/384-update-urls-on-new-banner-designs